Godeps

```diff
@@ -18,12 +18,15 @@ github.com/gogo/protobuf c57e439bad574c2e0877ff18d514badcfced004d
 github.com/golang/protobuf 2402d76f3d41f928c7902a765dfc872356dd3aad
 github.com/golang/snappy 723cc1e459b8eea2dea4583200fd60757d40097a
 github.com/gonuts/go-shellquote e842a11b24c6abfb3dd27af69a17f482e4b483c2
+github.com/gorilla/context 1c83b3eabd45b6d76072b66b746c20815fb2872d
+github.com/gorilla/mux 26a6070f849969ba72b72256e9f14cf519751690
 github.com/hailocab/go-hostpool 50839ee41f32bfca8d03a183031aa634b2dc1c64
 github.com/hashicorp/go-msgpack fa3f63826f7c23912c15263591e65d54d080b458
 github.com/hashicorp/raft b95f335efee1992886864389183ebda0c0a5d0f6
 github.com/hashicorp/raft-boltdb d1e82c1ec3f15ee991f7cc7ffd5b67ff6f5bbaee
 github.com/influxdata/influxdb 0e0f85a0c1fd1788ae4f9145531b02c539cfa5b5
-github.com/influxdb/influxdb 0e0f85a0c1fd1788ae4f9145531b02c539cfa5b5
+github.com/influxdb/influxdb c1d6c14c47dfb4d60878a979fbdd8b526281d3d6
+github.com/influxdb/telegraf d3a5cca1bc18f315dacbd9941f1e4556fde86538
```

Comment:
Not sure what happened here, but it shouldn't list itself as a dependency. What does your project source directory look like? is it not `$GOPATH/src/github.com/influxdata/telegraf`?